packages/mui-joy/src/Card/CardProps.ts

```diff
@@ -1,51 +1,68 @@
 import * as React from 'react';
 import { OverridableStringUnion, OverrideProps } from '@mui/types';
 import { ColorPaletteProp, VariantProp, SxProps, ApplyColorInversion } from '../styles/types';
+import { SlotProps, CreateSlotsAndSlotProps } from '../utils/types';
 
 export type CardSlot = 'root';
 
+export interface CardSlots {
```

Comment:
I assume that we can't export any of these types since there is no `slots` prop?

```suggestion
```

Reply:
@hbjORbj Is it necessary to export these interfaces (including other components)?

Reply:
For other components, yes, we need to export this to read the JSDoc and generated API docs. More specifically, `parseSlots` function needs it. https://github.com/mui/material-ui/blob/master/packages/api-docs-builder/utils/parseSlots.ts